src/app/youtube/directives/border2.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, OnInit, ElementRef, Renderer2, Input } from '@angular/core';
+
+@Directive({
+  selector: '[appBorder2]'
+})
+export class Border2Directive implements OnInit {
+  @Input() publishedAt = '';
+
+  constructor(private element: ElementRef, private renderer: Renderer2) { }
+  ngOnInit() {
+    let diff = Math.floor((new Date().getTime() - new Date(this.publishedAt).getTime()) / 86400000);
```

Comment:
same here and every where when magic number appears.

Reply:
I added constants